### PR TITLE
Update Portfile whizard 2.6.4 -> 2.8.2

### PR DESCRIPTION
**Pull request by Romendakil**
#### Description
whizard: update to version 2.8.2

* update to version 2.8.2
* remove obsolete patch file 

Closes: https://trac.macports.org/ticket/60004
Closes: https://github.com/macports/macports-ports/pull/6226

###### Type(s)

- [ ] bugfix
- [X] enhancement
- [ ] security fix

###### Tested on

macOS 10.15.2
Xcode 11.3.1

###### Verification 
Have you

- [x] followed our [Commit Message Guidelines](https://trac.macports.org/wiki/CommitMessages)?
- [ ] squashed and [minimized your commits](https://guide.macports.org/#project.github)?
- [x] checked that there aren't other open [pull requests](https://github.com/macports/macports-ports/pulls) for the same change?
- [x] referenced existing tickets on [Trac](https://trac.macports.org/wiki/Tickets) with full URL?

- [ ] checked your Portfile with `port lint`?
- [ ] tried existing tests with `sudo port test`?
- [ ] tried a full install with `sudo port -vst install`?
- [ ] tested basic functionality of all binary files?

### CODE REVIEW

File: science/whizard/Portfile [hunk 1]
```diff
@@ -29,14 +29,12 @@ long_description    WHIZARD is a program system designed for the efficient \
...
-checksums           rmd160  8793fcea26a40626cacb3a85fdb8e7a4ef67c0cf \
-                    sha256  58af39329f3445e36d0fbfea7010568cf3653a5b4d249cc1cd8ccfb04ceddfa5 \
-                    size    34683204
+checksums           rmd160  9eab0ed45a5e23b278430f07f3d1b835bc1e05b3 \
+                    sha256  32c9be342d01b3fc6f947fddce74bf2d81ece37fb39bca1f37778fb0c07e2568 \
+                    size    35344103
 
 depends_build       port:ocaml
 
-patchfiles          WOXDR.cpp.patch
```
Comment (mf2k) [inline science/whizard/Portfile]: Please remove this patchfile too.

**Comment by mf2k** [comment]
Please update the commit message to conform to the guidelines.

**Comment by mf2k** [comment]
The commits just need to be squashed into 1 and the message improved.

**Comment by ryandesign** [comment]
> Port whizard's dependencies **fail** on xcode9.4. [Log](https://paste.macports.org/923f5c350bcc)

This failed due to Travis' typical intermittent network problems which are out of our control.

**Comment by ryandesign** [comment]
> The commits just need to be squashed into 1 and the message improved.

@mf2k Which we can do using the "Squash and merge" button.

@Romendakil Thank you!

**Pull request merged**